extensions/wakatime/package.json

```diff
@@ -16,47 +16,66 @@
       "subtitle": "WakaTime",
       "title": "Show Summary",
       "description": "Show your WakaTime Summary",
-      "mode": "view"
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "apiKey",
+          "description": "Use your WakaTime API key to use your account",
+          "type": "password",
+          "required": true,
+          "title": "WakaTime API Key"
+        }
+      ]
     },
     {
       "name": "leaderboard",
       "subtitle": "WakaTime",
       "title": "Show Public Leaderboard",
-      "description": "Show your rank in the public global leaderboard",
+      "description": "An overview of the public leaderboard. Check your rank against others all around the world.",
```

Comment:
Thoughts on this?

Reply:
`Check your rank against other all around the world.` should be enough 😸